babel.config.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function(api) {
```

Comment:
What a mess... wow. Any chance of adding some explanatory comments?

Reply:
This was, again, autogenerated by rails. I don't know why babel is configured like it is and thus can only guess what it does.

But I don't recall having touched this in last year's project, so maybe it's just fine if we say something like "This is the configuration for babel <link to wiki what babel is>. For further information look at <config documentation>"?